### ISSUE TITLE
Sprite visibility not honored between timelines

### ISSUE THREAD

**Issue by dillon-courts** [spritebuilder/SpriteBuilder#670]
- Create a blank spritebuilder project. 
- Add a sprite and uncheck the visibility box. 
- Do not add any keyframes, signifying you want the sprite to not be visible during this timeline. 
- Create another timeline where you set the visibility keyframe for the duration of the timeline. 
- If you run the timeline with the visibility keyframe set and then run the timeline without the keyframe set, the sprite will remain visible.

Basically a CCSprite does not honor its visibility keyframe when it is toggled off again after it was on in a separate timeline. This may apply to all CCNode subclasses but I only tested a sprite.

Here is a sample project that displays the issue: https://dl.dropboxusercontent.com/u/966509/testVisibility.spritebuilder.zip

**Comment by cocojoe** [comment]
Turns out this isn't really supported however no reason it shouldn't be.  I made a patch that worked, just running some more tests.
